### PR TITLE
add module openedge-function-python36 provides the Python36 function runtime

### PR DESCRIPTION
**Pull request by chensheng0**
Add module openedge-function-python36 provides the Python36 function runtime, which can be dynamically started by openedge-function-manager

### CODE REVIEW

File: example/docker/var/db/openedge/function-sayhi3-code/sayhi3.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
```
Comment (kuailehaibin) [inline example/docker/var/db/openedge/function-sayhi3-code/sayhi3.py]: #!/usr/bin/env python --> #!/usr/bin/env python3.6

File: openedge-function-python36/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+FROM base as builder
+RUN apk add --update --no-cache \
+    gcc \
+    linux-headers \
+    make \
+    musl-dev \
+    python-dev \
+    g++
+RUN python -m pip install --upgrade pip
+RUN pip install --install-option="--prefix=/install" pyyaml protobuf grpcio
```
Comment (kuailehaibin) [inline openedge-function-python36/Dockerfile]: pyyaml 不需要

File: example/docker/var/db/openedge/application.yml [hunk 3]
```diff
@@ -62,6 +62,16 @@ services:
       - name: function-sayhi-code
         path: var/db/openedge/function-sayhi
         readonly: true
+  - name: function-sayhi3
+    image: openedge-function-python36
+    replica: 0
+    mounts:
+      - name: function-sayhi3-conf
```
Comment (ludanfeng) [inline example/docker/var/db/openedge/application.yml]: 可以和function-sayhi公用配置的吧

File: example/docker/var/db/openedge/application.yml [hunk 4]
```diff
@@ -62,6 +62,16 @@ services:
...
+        path: etc/openedge
+        readonly: true
+      - name: function-sayhi3-code
```
Comment (ludanfeng) [inline example/docker/var/db/openedge/application.yml]: 可以和function-sayhi公用代码吗？

File: example/docker/var/db/openedge/function-sayhi3-code/sayhi3.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    if 'c' in event:
+        while True:
+            pass
+
+    if 'invoke' in event:
+        res = context.invoke(event['invoke'], event['invokeArgs'])
+        res['invoked'] = True
+        return res
+
+    event['functionName'] = context['functionName']
```
Comment (ludanfeng) [inline example/docker/var/db/openedge/function-sayhi3-code/sayhi3.py]: 这里帮忙优化一下代码，先判断字段存不存在，再取值

**Pull request merged**